opentech/public/utils/models.py

```diff
@@ -246,6 +246,11 @@ class Meta:
     ]
 
 
+@register_setting
+class ComplianceSettings(BaseSetting):
```

Comment:
This isnt part of the public site. It should exist under apply.

Also - we probably want to call it `ProjectSettings` so we can add the finance email as well.

Reply:
Fixed and rebased.